.github/actions/scan-docker-image/action.yml

```diff
@@ -0,0 +1,27 @@
+name: Trivy Scan Action
+description: Scan Docker images using Trivy and upload results.
+inputs:
+  image:
+    description: A docker image
+    required: true
+  category:
```

Comment:
let's remove `category` and use `image` for `sarif` category.

Reply:
Okay